lib/scheduler.cc

```diff
@@ -55,306 +57,305 @@ void Worker::work() const {
+void Worker::execute_reaction(Reaction *reaction) const {
+  log::Debug() << "(Worker " << identity_ << ") "
                << "execute reaction " << reaction->fqn();
+
   tracepoint(reactor_cpp, reaction_execution_starts, id, reaction->fqn(),
              scheduler.logical_time());
   reaction->trigger();
   tracepoint(reactor_cpp, reaction_execution_finishes, id, reaction->fqn(),
              scheduler.logical_time());
 }
 
-void Scheduler::schedule() {
+void Scheduler::schedule() noexcept {
   bool found_ready_reactions = schedule_ready_reactions();
 
   while (!found_ready_reactions) {
     log::Debug() << "(Scheduler) call next()";
     next();
-    reaction_queue_pos = 0;
+    reaction_queue_pos_ = 0;
 
     found_ready_reactions = schedule_ready_reactions();
 
-    if (!continue_execution && !found_ready_reactions) {
-      // let all workers know that they should terminate
+    if (!continue_execution_ && !found_ready_reactions) {
+      // let all workers_ know that they should terminate
       terminate_all_workers();
       break;
     }
   }
 }
 
-Reaction* ReadyQueue::pop() {
-  auto old_size = size.fetch_sub(1, std::memory_order_acq_rel);
+auto ReadyQueue::pop() -> Reaction * {
+  auto old_size = size_.fetch_sub(1, std::memory_order_acq_rel);
 
   // If there is no ready reaction available, wait until there is one.
   while (old_size <= 0) {
     log::Debug() << "(Worker " << Worker::current_worker_id()
                  << ") Wait for work";
-    sem.acquire();
+    sem_.acquire();
     log::Debug() << "(Worker " << Worker::current_worker_id() << ") Waking up";
-    old_size = size.fetch_sub(1, std::memory_order_acq_rel);
+    old_size = size_.fetch_sub(1, std::memory_order_acq_rel);
     // FIXME: Protect against underflow?
   }
 
   auto pos = old_size - 1;
-  return queue[pos];
+  return queue_[pos];
 }
 
-void ReadyQueue::fill_up(std::vector<Reaction*>& ready_reactions) {
-  // clear the internal queue and swap contents
-  queue.clear();
-  queue.swap(ready_reactions);
+void ReadyQueue::fill_up(std::vector<Reaction *> &ready_reactions) {
+  // clear the internal queue_ and swap contents
+  queue_.clear();
+  queue_.swap(ready_reactions);
 
   // update the atomic size counter and release the semaphore to wake up
   // waiting worker threads
-  std::ptrdiff_t new_size = queue.size();
-  auto old_size = size.exchange(new_size, std::memory_order_acq_rel);
+  auto new_size = static_cast<std::ptrdiff_t>(queue_.size());
+  auto old_size = size_.exchange(new_size, std::memory_order_acq_rel);
 
-  // calculate how many workers to wake up. -old_size indicates the number of
-  // workers who started waiting since the last update.
-  // We want to wake up at most all the waiting workers. If we would release
-  // more, other workers that are out of work would not block when acquiring
+  // calculate how many workers_ to wake up. -old_size indicates the number of
+  // workers_ who started waiting since the last update.
```

Comment:
names shouldn't change in the comments. (This goes for all the comments  in this file)